snowflake_queries/snowflake_unusual_logins.yml

```diff
@@ -0,0 +1,16 @@
+AnalysisType: scheduled_rule
+Filename: snowflake_unusual_logins.py
+RuleID: Snowflake.UnusualLogins
```

Comment:
This makes me think of the standard "Unusual login" detection. Wonder if we could rename to something else? 🤷🏻‍♀️

Reply:
Yeah thats a good call, I changed it to "Unusual login volume"